widget/hyperlink_test.go

```diff
@@ -24,15 +24,16 @@ func TestHyperlink_MinSize(t *testing.T) {
 }
 
 func TestHyperlink_Alignment(t *testing.T) {
-	hyperlink := &Hyperlink{Text: "Test", Alignment: fyne.TextAlignTrailing}
```

Comment:
Again, why are these tests being changed? it's a warning sign

Reply:
See comment below